packages/cfpb-expandables/src/Expandable.js

```diff
@@ -47,6 +47,8 @@ function initialize() {
     this.ui.content
   );
   this.transition = transition.init();
+  this.transition.addEventListener( 'expandBegin', expandBeginHandler.bind( this ) );
```

Comment:
I don't believe this can go in the `events` further up in this file because it is on the transition, but if someone knows better let me know.